common/test/acceptance/pages/studio/import_export.py

```diff
@@ -115,6 +116,29 @@ class ExportLibraryPage(ExportMixin, TemplateCheckMixin, LibraryLoader, LibraryP
     """
     Export page for Libraries
     """
+    def get_nav_help_css(self):
+        """
+        Returns the unique css of nav help DOM element
+        """
+        return NAV_HELP_CSS
```

Comment:
I don't think you need this method if you are importing it above. You an just use `NAV_HELP_CSS`